Pegasus_Embedding_Video/embed_draw_sparse.py

```diff
@@ -12,20 +12,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from pegasus_graph import P6
-import dwave_networkx as dnx, networkx as nx
+from pegasus_graph import P16, P6
+import minorminer.layout as mml, dwave_networkx as dnx, networkx as nx
+import matplotlib
+matplotlib.use("agg")
 from matplotlib import pyplot as plt, colors as mpl_color
 
-from minorminer import layout
+# Draw a small P6 graph
+n = 200
+C = nx.random_regular_graph(3, n)
 
-nodes = 200
-degree = 3
-C = nx.random_regular_graph(degree, nodes)
+emb, (layout_C, layout_P) = mml.find_embedding(C, P6, random_seed = 1,
+                                                    return_layouts = True,
+                                                    threads = 3)
 
-layout_args = {'random_seed': 1, 'return_layouts': True}
+plt.figure(figsize=(20, 20))
 
-emb, (layout_C, _) = layout.find_embedding(C, P6, **layout_args)
+nx.draw(C)
 
-nx.draw(C, pos = layout_C)
+plt.savefig("sparse_graph.png")
+plt.close()
 
+plt.figure(figsize=(20, 20))
 dnx.draw_pegasus_embedding(P6, emb, C)
+plt.savefig("sparse_embedded.png")
+plt.close()
+
+# Draw a large P16 graph (this will take a while!)
+if False:
```

Comment:
Don't think we need to do it for this PR but maybe these should be moved into functions